rebar.config

```diff
@@ -74,6 +74,7 @@
     , {esasl, {git, "https://github.com/emqx/esasl", {tag, "0.2.0"}}}
     , {jose, {git, "https://github.com/potatosalad/erlang-jose", {tag, "1.11.2"}}}
     , {telemetry, "1.1.0"}
+    , {hackney, {git, "https://github.com/benoitc/hackney", {tag, "1.18.1"}}}
```

Comment:
Q: what this for, does anywhere used the `hackney`

Reply:
Yes, hackney is used by clickhouse.